patches/chromium/desktop_media_list.patch

```diff
@@ -7,10 +7,20 @@ Subject: desktop_media_list.patch
 * Free the one-time use capturer after thumbnails are fetched
 
 diff --git a/chrome/browser/media/webrtc/desktop_media_list.h b/chrome/browser/media/webrtc/desktop_media_list.h
-index a489bf6ea2179059f53e53563e993db9c7cd123b..8d3df446290e2111e4952899539cbc16d37546e5 100644
+index a489bf6ea2179059f53e53563e993db9c7cd123b..93e237569fe94ec8526f67d915e1a7352d586b16 100644
 --- a/chrome/browser/media/webrtc/desktop_media_list.h
 +++ b/chrome/browser/media/webrtc/desktop_media_list.h
-@@ -69,6 +69,7 @@ class DesktopMediaList {
+@@ -38,6 +38,9 @@ class DesktopMediaList {
+ 
+   virtual ~DesktopMediaList() {}
+ 
++  // Allows listening to notifications generated by the model.
++  virtual void AddObserver(DesktopMediaListObserver* observer) = 0;
```

Comment:
bleh, can we upstream this or find another way to get this info?

Reply:
Yeah I will try that as part of https://github.com/electron/electron/issues/20771